crates/aiken-lang/src/builtins.rs

```diff
@@ -25,6 +26,16 @@ use uplc::{
 pub const PRELUDE: &str = "aiken";
 pub const BUILTIN: &str = "aiken/builtin";
 
+pub static INTERNAL_FUNCTIONS: LazyLock<BTreeSet<&'static str>> = LazyLock::new(|| {
+    let mut set = BTreeSet::new();
+    set.insert("diagnostic");
+    set.insert("do_from_int");
+    set.insert("encode_base16");
+    set.insert("enumerate");
+    set.insert("from_int");
+    set
+});
+
```

Comment:
Kind of unrelated to the change in the PR itself, but I stumble upon this while debugging / testing. Somehow, we kept polluting all module exported identifiers with the default internal functions used in the prelude and elsewhere. 

Which seemed utterly wrong, so I cleaned them up. In **theory**, this shouldn't cause any issue (and might event be quite beneficial overall as we'll be cloning and duplicating a lot less data).

Reply:
yea makes sense